machine-runner/README.md

```diff
@@ -279,6 +279,49 @@ warehouse.events.on('error', (error) => {
 })
 ```
 
+#### Event List
+
+##### `next`
+
+A `next` event is emitted when a state transition happens and the machine runner has processed all events matching the supplied tag.
+
+The payload is `StateOpaque`, similar to the value produced in the `for-await` loop. 
+
+##### `error`
+
+An `error` event is emitted when an error happened inside the runner. Currently this is the list of the errors:
+- A command is called when locked i.e. another command is being issued in the same machine
```

Comment:
Perhaps in addition to the problem statement, explain that after invoking a command the app needs to await the resulting Promise to see whether it was successful (in which case it needs to await the next state update) or unsuccessful (in which case it can try again).

Reply:
This explanation should go to another section. Yet, this part of our API is not mature yet.
Currently, a failing command does not throw.
In this version, a failing command returns `Promise<void>`, while a successful one returns `Promise<Metadata>`.
I'll open an issue to decide about this and put this on version 0.4.6

Reply:
Will address this in https://github.com/Actyx/machines/issues/71